### ISSUE TITLE
Isn't it a pain in the ass to operate if you have many outlook accounts?

### ISSUE THREAD

**Issue by szsyzx** [blaine-t/fwd_outlook_graph#1]
Isn't it a pain in the ass to operate if you have many outlook accounts?

**Comment by plumpastry** [comment]
agreed, do you want to fork the repository so we can work on a fix?

**Comment by szsyzx** [comment]
Do you have real-time chat software? I would like to inquire about specific deployment methods
